process.py

```diff
@@ -279,14 +284,30 @@ def issue_lookup(jira, release, version, packages, zstream):
     try:
         issues = jira.search_issues(jql_query)
     except JIRAError as e:
-        print(e);
+        print(e)
 
     if len(issues) != 1:
         print(f'Found {len(issues)} issues matching {summary}')
         return "0", None
 
     return issues[0].key, issues[0];
 
+def issue_request_clone(jira, release, version, packages):
+    package = packages.split(',')[0]
+    summary=bug_summary_short%year
+
+    _, issue = issue_lookup(jira, release, version, packages, False)
+    if issue == None:
+        return "0", None
```

Comment:
The other return code returns a bool, did you really intend to return ("0", None) here. Won't that stilll parse as 'true' in an if?